src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -737,7 +737,7 @@
       },
       {"id": 121,
         "name": "dialect_options.state_machine_options.delimiter",
-        "type": "CSVOption<char>"
+        "type": "CSVOption<string>"
```

Comment:
This is done in e.g. this PR - https://github.com/duckdb/duckdb/pull/14043/files#diff-b05d9518b51946935610619b9972b5349971a9f9d25d70f22265425d40188a73

Reply:
I think we can just rename the nested properties to `"delimiter"`. Technically that breaks JSON (de)serialization - but since this is at the logical plan level I don't think that is being actively used anywhere.